cookbooks/bcpc-hadoop/templates/default/ooz_oozie-site.xml.erb

```diff
@@ -278,6 +278,16 @@
     <!-- Default proxyuser configuration for Hue -->
 
   <% if node[:bcpc][:hadoop][:kerberos][:enable] == true then %> 
+
+  <property>
+    <name>oozie.credentials.credentialclasses</name>
+    <value>
+      hcat=org.apache.oozie.action.hadoop.HCatCredentials,
+      hbase=org.apache.oozie.action.hadoop.HbaseCredentials,
```

Comment:
What happens if one of these classes can not be found? I am particularly thinking of the composability of our clusters where we may not have HBase or Hive installed on a cluster but this is hardcoded. (If things bomb out, then it seems future work could make this discoverable.)

Further, since HBase will come in a future PR, can that be saved until full HBase Oozie support is added?